src/Uno.UWP/ApplicationModel/DataTransfer/Clipboard.wasm.cs

```diff
@@ -1,21 +1,159 @@
 ﻿#nullable disable // Not supported by WinUI yet
 
 using System;
-using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
-using Windows.UI.Core;
-using Uno.Extensions.Specialized;
+using Uno.Disposables;
 using Uno.Foundation;
-using System.Threading;
+using Uno.Helpers.Serialization;
+using Windows.Storage.Streams;
 
 namespace Windows.ApplicationModel.DataTransfer
 {
 	public static partial class Clipboard
 	{
 		private const string JsType = "Uno.Utils.Clipboard";
 
+		[DataContract]
+		private class MarshalledData: IDisposable
```

Comment:
Move this type to a Clipboard.MarshalledData.wasm.cs for readability. You can keep it as a nested type.

Reply:
Done.